apps/etterlatte-fordeler/.nais/prod.yaml

```diff
@@ -38,7 +38,12 @@ spec:
       value: http://etterlatte-pdltjenester/person
     - name: PDL_AZURE_SCOPE
       value: api://prod-gcp.etterlatte.etterlatte-pdltjenester/.default
+    - name: DIGDIRR_KRR_URL
+      value: https://digdir-krr-proxy.intern.nav.no
+    - name: DIGDIR_KRR_AZURE_SCOPE
+      value: api://prod-gcp.team-rocket.digdir-krr-proxy/.default
   accessPolicy:
     outbound:
       rules:
-        - application: etterlatte-pdltjenester
\ No newline at end of file
+        - application: etterlatte-pdltjenester
+        - application: digdir-krr-proxy
```

Comment:
Har du husket å be dem til å legge til oss som inbound også?

Reply:
Jepp, de har lagt oss inn https://github.com/navikt/digdir-krr/issues/201